legit_test.py

```diff
@@ -0,0 +1,71 @@
+import legit
```

Comment:
we will end up with many tests. I suggest to create a tests dir right away and move test code there. Please also name your test module according to the pytest convention

Reply:
[> In those directories, search for test_*.py or *_test.py files, imported by their test package name.](http://doc.pytest.org/en/latest/goodpractices.html)

I suppose in this case you insist on test_*.py?

Reply:
Anyway I've renamed file, I suggest mention this rule is style conventions